File: Laboratorio2/Chandu_and_his_Girlfriend_Returns.cpp
    #include <stdio.h>

    using namespace std;

    void llenado(int ar[],int tam){
            for(int i=0;i<tam;i++){
                int elemento;
                scanf("%d",&elemento);
                ar[i] = elemento;
            }
        }

    int main() {
        	int casos;
        	scanf("%d",&casos);
        	for(int i=0;i<casos;i++){
        	    int arr,arr2;
        	    scanf("%d",&arr);
        	    scanf("%d",&arr2);
        	    int ar[arr];
        	    llenado(ar,arr);
        	    int cont=0;
        	    for(int j=0;j<arr2;j++){
                    int num;
                    scanf("%d ",&num);
                    if(num>ar[cont]){
                        printf("%d ",num);
                    }else{
                        while(num<ar[cont] && (cont<arr)){
                            printf("%d ",ar[cont]);
                            cont++;
                        }
                        printf("%d ",num);
                    }
                }
                while(cont<arr){
                    printf("%d ",ar[cont]);
                    cont++;
                }
                printf("\n");

            }

        }

File: Laboratorio2/Chandu_and_his_Girlfriend _Returns .py
casos = int(input())
for i in range(casos):
    tam = list(map(int,input().split()))
    lista1 = list(map(int,input().split()))
    lista2 = list(map(int,input().split()))
    cont = 0
    for j in range(tam[1]):
        if cont<tam[0]:
            if lista2[j]>lista1[cont]:
                print(lista2[j],end=" ")
            else:
                while lista2[j] < lista1[cont]:
                        print(lista1[cont],end=" ")
                        cont += 1
                        if cont==tam[0]:
                             break
                print(lista2[j],end=" ")
        else:
            print(lista2[j],end=" ")

    while(cont<tam[0]):
        print(lista1[cont],end=" ")
        cont +=1
    print()